docs/sdks/sdk-ref-python.md

```diff
@@ -209,6 +209,8 @@ If you're using server-side integration (see [Publisher Integration Guide, Serve
    If the user has opted out, this method returns `None`, indicating that the user's identity should be removed from the session. To confirm optout, you can use the `token_refresh_response.is_optout()` function.
 
 ## Usage for Advertisers/Data Providers
```

Comment:
There needs to be some copy between these two headings.

Reply:
added a line saying there are two operations

Reply:
thx. Just adjusted ending punctuation.